triton-vm/src/table/table_collection.rs

```diff
@@ -67,49 +75,135 @@ fn to_vec_vecs<T: Sized + Clone, const S: usize>(vector_of_arrays: &[[T; S]]) ->
         .collect_vec()
 }
 
+pub fn interpolant_degree(padded_height: usize, num_trace_randomizers: usize) -> Degree {
+    (padded_height + num_trace_randomizers - 1) as Degree
+}
+
+pub fn derive_omicron<DataPF: FiniteField>(padded_height: u64) -> DataPF {
+    debug_assert!(
+        0 == padded_height || is_power_of_two(padded_height),
+        "The padded height was: {}",
+        padded_height
+    );
+    DataPF::primitive_root_of_unity(padded_height).unwrap()
+}
+
+/// Returns the relation between the FRI domain and the omicron domain
+pub fn unit_distance(padded_height: usize, omega_order: usize) -> usize {
```

Comment:
Since `padded_height` is never zero, I don't think this function adds anything. We could simply inline the division `omega_order / padded_height` in every place where this function is invoked, ad put the result of that division into a variable named so as to describe its purpose.

Reply:
Fixed in cd1c8e9572c0c855821563b7d0a0b6086f9b1813